sync/class.jetpack-sync-module-callables.php

```diff
@@ -86,6 +86,11 @@ public function maybe_sync_callables() {
 		if ( get_transient( self::CALLABLES_AWAIT_TRANSIENT_NAME ) ) {
 			return;
 		}
+
+		if ( ! defined( 'REST_API_REQUEST' ) ) {
+			define( 'REST_API_REQUEST', true );
```

Comment:
An alternative would be to remove the check for `REST_API_REQUEST` from our existing implementations. Not 100% sure why it's there, since the performance benefit would be negligible.